src/RVRP_data.jl

```diff
@@ -1,9 +1,10 @@
 struct TimeWindow
-    begin_time::Float64
-    end_time::Float64
+    opening_date::Float64
+    soft_closing_date::Float64 # must be greater or equal to the opening_date
+    hard_closing_date::Float64 # must be greater or equal to the soft_closing_date
 end
 
-mutable struct OperationPoint # can be a Depot, Pickup, Delivery, Recharging, ..., or a combination of those
+mutable struct Location # Location where can be a Depot, Pickup, Delivery, Recharging, ..., or a combination of those services
```

Comment:
[Subjective] OperationPoint sounds to me more like LogicalLocation, while Location sounds more like PhysicalLocation. the latter can be awkward when splitting based on opening times etc. I'm very fine with Location though.